Sources/SmithyXML/ReadingClosures.swift

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.Date
+import typealias SmithyReadWrite.ReadingClosure
+import enum SmithyTimestamps.TimestampFormat
```

Comment:
This file provides reading closures for many Smithy model types that do not have their own code-generated closure.